bancorprotocol/carbon-app
# Concentrated Liquidity range chart

fix #1442

Add responsive graph into trade

What to test : 

A. Trade
1. go to trade, create concentrated liquidity
2. resize the screen from mobile to desktop
3. zoom in & out of the graph
4. move cursor around
5. change market price -> title changes
6. use PARQ token (not market price)
7. create the strategy

B. Edit No market price
1. go to edit of this strategy
2. Chart should be disabled with "No Market Price found"
3. Enter a price
4. Chart should show "User market price" and is available

C. Edit with market price
1. go to edit of an overlapping strategy market price
2. marginal price is the same as in the previous state
3. after updated, the strategy marginal price is updated
